File: fetch_perms.js
j = "perms = {"
for(var row of document.querySelector("h6#permissions-bitwise-permission-flags + table > tbody").children) {
    perm = row.children[0].textContent;
    j += `\n`;
    j += `    "${perm.endsWith(" *") ? perm.slice(0, -2) : perm}":`.padEnd(35);
    j += `${row.children[1].textContent.split(' ')[0]}n,`;
}
j = j.slice(0, -1) + "\n}";
console.log(j)

File: fetch_perms.py
print("perms = {"
for x in range(53):
    perms = discord.Permissions(1 << x)
    for perm in perms:
        if perm[1]:
            print("   ", f'"{perm[0].upper()}":'.ljust(25), f"0x{1 << x:08x},")
print("}")
